### ISSUE TITLE
A11y_AzureDataStudio_Notebooks - Run_Resize:On 200 % Zoom, Some of the content like Sort Ascending and Sort Descending in the Filter dialog is not completely visible.

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio#22655]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev1; #DesktopApp; #Win32; #WCAG1.4.4; #Resize; #Benchmark; #Zoom; #AILimited; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; 

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**
### Environment and OS details:
Azure Data Studio Version: 1.42.0 (user setup)
Windows 11 Enterprise Version	22H2
Installed on	‎28-‎12-‎2022
OS build	22621.1413
Experience	Windows Feature Experience Pack 1000.22639.1000.0
### Pre-Requisite:

1. Install Azure data studio From here https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall
2. Once installation done , Install MySQL Extension in Azure data studio
3. Tab Till  new connection and press enter 
4. Enter a valid server name username and password and press connect
### Pre-Requisite For Resize:

1. Open Setting, Tab till System enter and Tab till display press enter
2.  Set the machine’s target resolution to an "average" mid-range resolution (resolution)
3. Zooming: Drag the text size slider to 200% (or) Scaling: apply advanced scaling to 200       

### Repro Steps:

1. Launch Azure Data Studio.
2. Open an existing notebook that has code cells in it.
3. Click Run Cells on Notebook toolbar.
4. Tab till Filter
5. Apply Resize
6. Verify that On 200% Zoom Filter dialog is completely visible or not.
### Actual Result:
On 200 % Zoom, Some of the content like Sort Ascending and Sort Descending in the Filter dialog is not completely visible.
### Expected Result:
On 200 % Zoom, All the content in the Filter dialog should be completely visible.
### Attachment:
**At 100 % Zoom:**
![100 zoom filter dialog](https://user-images.githubusercontent.com/100189407/229021928-2d690562-f04f-4e02-8721-86b04b39c488.png)
**Zooming Method:**
![Zooming Method](https://user-images.githubusercontent.com/100189407/229023003-48713f4a-02bb-41c9-a855-53d7677c3dbb.gif)
**Scaling Method:**
![Scaling method](https://user-images.githubusercontent.com/100189407/229023303-951285cd-37f1-403c-bb5e-346c17b30ca2.png)

**Comment by VijayaBharathyA** [comment]
Primary BUG :https://github.com/microsoft/azuredatastudio-mysql/issues/191

**Comment by VijayaBharathyA** [comment]
#Regressed:05-15-23; #Closed

### Verified issue in below environment :
Issue is fixed
Version: 1.45.0-insider (user setup)
Commit: https://github.com/microsoft/azuredatastudio/commit/4329d04da3b86689556d4dceab7d9b3d5cde3f58
Date: 2023-05-15T05:30:26.887Z
VS Code: 1.70.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621![image](https://github.com/microsoft/azuredatastudio/assets/100189407/256206f5-bd8b-45ea-bdea-1619c592b860)
